strings/antagonist_flavor/thief_flavor.json

```diff
@@ -0,0 +1,38 @@
+{
+	"Black Market Outfitter": {
+		"goal": "You've tapped into the black market, and it turns out you could get rich quick introducing some of Nanotrasen's guns to the dark web!",
+		"introduction": "You're a Black Market Outfitter.",
+		"objective_needs_target": 0,
+		"objective_type": "/datum/objective/steal_n_of_type/summon_guns"
```

Comment:
Brother why do you gotta do me like this, string files are for strings, not typepaths or bools

Reply:
alright, i have a good idea on how to move these off the json.